index.html

```diff
@@ -2494,34 +2522,90 @@
             <code>"web-nfc-only"</code>, and <var>messageId</var> is not
```

Comment:
It's initialized at the beginning of the steps of a different algorithm. We don't use dynamic scoping in web algorithms. You need to pass in whatever variables you intend to use.